client/signup/config/flows.js

```diff
@@ -58,13 +58,17 @@ function getThankYouNoSiteDestination() {
 function getChecklistThemeDestination( dependencies ) {
 	return `/checklist/${ dependencies.siteSlug }?d=theme`;
 }
+function getEditorDestination( dependencies ) {
+	return `/block-editor/page/${ dependencies.siteSlug }/home`;
```

Comment:
What do you think about appending the `?new-homepage=1` query param here? That will automatically open the Homepage template selector, which might make sense in this context.

Reply:
I like the idea of adding in this query param and getting people to select a template for the homepage, but without the user clicking the button to update their homepage, does the Homepage template selector provide enough context to what they're meant to do there? And once we've added the theme step in the `test-fse` flow, without a heading on this template page to say that it's just for the homepage, will users feel like they're having to choose a theme twice?

Reply:
I think adding the query parameter should be done based on whether or not the user sees the theme step or a preview in Signup. If either of those are true, then they're already expecting some sort of default content.

Reply:
Also if they see the theme step but choose to skip.